easybuild/easyconfigs/r/ruffus/ruffus-2.8.4-foss-2021b.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'ruffus'
+version = '2.8.4'
+
+homepage = 'http://www.ruffus.org.uk/'
+description = "Ruffus is a Computation Pipeline library for python. It is open-sourced, powerful and user-friendly, and widely used in science and bioinformatics."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['6cd3d96e459a1aedcc10341f14fddedfdc13664551757fcd429a241de63826aa']
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('Graphviz', '2.50.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+sanity_check_commands = [
+    "python -c 'import ruffus'",
```

Comment:
This `import` is already done by default, so drop it and only do the import check on `ruffus.graph`:

```python
sanity_check_commands = ["python -c 'import ruffus.graph'"]